participants/maheshwaran_velusamy.json

```
@@ -0,0 +1,53 @@
+// vim: ft=jsonc
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Maheshwaran",
+		"familyName": "Velusamy",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": false,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": false
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "mahes1287",
+	// company name (optional)
+	"company": "Veact GmbH",
+	// either both days or at least one day has to be set to true.
+	// If you cannot make one of the days, please set it to false, to allow someone else take that spot!
+	"when": {
+		// June 27th, 2025
+		"friday": true,
+		// June 28th, 2025
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": false,
+	// your current interests (JS and in general) (required)
+	"tags": ["Svelte", "dart", "Postgres", "MongoDB"],
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": false,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	"allergies": [],
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "I always amazed how its run the web ecosystem and beyond with lot of crtiticism",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "I am willing to assist in any way possible during the event.",
+	// if you want a T-Shirt we need your size and variant preference (optional)
+	// the following sizes are available: S, M, L, XL, 2XL, 3XL (only regular cut)
+	"tShirtSize": "M",
+	// your Mastodon URL (optional)
+	"mastodon": "",
+	// your LinkedIn profile URL
+	"linkedin": "https://www.linkedin.com/in/maheshwaranv/",
+	// your X (Twitter) handle - must match regex ^[a-zA-Z_]{1}[a-zA-Z0-9_]{0,14}$ (optional)
+	"X": "mahes1287",
+	// your website URL or other social media (optional)
+	"website": "https://www.maheshthedev.com"
```

Hey @mahes1287 , great to have you again at the JSCC!

Here is a suggestion to quickly fix your PR. Please merge and we can finish your registration:

```suggestion
	"website": "https://maheshthedev.com"
```

---

Thank you